src/wizards/tokenSwapDealWizard/components/daoStageContent/daoStageContent.ts

```diff
@@ -1,19 +1,65 @@
 import { IDAO } from "entities/DealRegistrationTokenSwap";
 import { availableSocialMedias } from "../../dealWizardTypes";
-import { bindable, inject } from "aurelia";
+import { bindable, inject, observable } from "aurelia";
 import { IValidationController } from "@aurelia/validation-html";
+import { FirestoreService } from "services/FirestoreService";
+import { IDAOsData } from "services";
+import { ethers } from "ethers";
+import { EnsService } from "services";
+import { Utils } from "services/utils";
+import { AutoCompleteSelect } from "cl-webcomp-poc/components/AutocompleteSelect";
+
+interface IAutocompleteSelectItem {
+  name: string,
+  avatarUrl?: string,
+  id: string,
+  treasury?: string[],
+}
+
+interface IAutocompleteSelectEvent extends Event {
+  detail: IAutocompleteSelectItem
+}
```

Comment:
`AutoCompleteSelect`
vs
`IAutocompleteSelectEvent`

For the future, would be nice to align on one naming convention (upper "Complete" or lower "complete")

Reply:
Refactored to have only `AutoCompleteSelect` everywhere.